src/scss/components/form/_index.scss

```diff
@@ -151,5 +151,27 @@ $form-asterisk-offset: $form-asterisk-size + $form-asterisk-gap;
   }
 }
 
+.hxErrorText {
+  color: $red-900;
+  font-size: 12px;
+  margin-left: inherit;
+}
+
```

Comment:
Please revert all changes to this file.  This will be handled in `src/scss/components/email/_index.scss`.

Reply:
Changes have been done, please review.